lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -163,64 +162,58 @@ def test_enabled_block_types(self, course_options, expected_blocks):
         block = TodaysDate(self.course, self.user)
         self.assertTrue(block.is_enabled)
         self.assertEqual(block.date, datetime.now(utc))
-        self.assertEqual(block.title, 'Today is {date}'.format(date=expected_display_date))
-        self.assertNotIn('date-summary-date', block.render())
-
-    @freeze_time('2015-11-01 08:59:00')
-    def test_todays_date_time_zone_daylight(self):
-        """
-        Test today's date block displays correctly during
-        daylight savings hours
-        """
-        self._today_date_helper('Nov 01, 2015 (01:59 PDT)')
-
-    @freeze_time('2015-11-01 09:00:00')
-    def test_todays_date_time_zone_normal(self):
-        """
-        Test today's date block displays correctly during
-        normal daylight hours
-        """
-        self._today_date_helper('Nov 01, 2015 (01:00 PST)')
+        self.assertEqual(block.title, 'current_datetime')
 
     @freeze_time('2015-01-02')
-    def test_todays_date_render(self):
+    def test_todays_date_no_timezone(self):
         self.setup_course_and_user()
-        block = TodaysDate(self.course, self.user)
-        self.assertIn('Jan 02, 2015', block.render())
+        self.client.login(username='mrrobot', password='test')
+
+        html_elements = [
+            '<h1 class="handouts-header">Important Course Dates</h1>',
+            '<div class="date-summary-container">',
+            '<div class="date-summary date-summary-todays-date">',
+            '<h3 class="heading localized-datetime"',
+            'data-datetime="2015-01-02 00:00:00+00:00"',
+            'data-string="Today is {date}"',
+            'data-timezone="None"'
+        ]
+        url = reverse('info', args=(self.course.id, ))
+        response = self.client.get(url)
+        for html in html_elements:
+            self.assertContains(response, html)
 
     @freeze_time('2015-01-02')
-    def test_todays_date_render_time_zone(self):
+    def test_todays_date_timezone(self):
         self.setup_course_and_user()
+        self.client.login(username='mrrobot', password='test')
         set_user_preference(self.user, "time_zone", "America/Los_Angeles")
-        block = TodaysDate(self.course, self.user)
-        # Today is 'Jan 01, 2015' because of time zone offset
-        self.assertIn('Jan 01, 2015', block.render())
+        url = reverse('info', args=(self.course.id,))
+        response = self.client.get(url)
 
-    ## CourseStartDate
+        html_elements = [
+            '<h1 class="handouts-header">Important Course Dates</h1>',
+            '<div class="date-summary-container">',
+            '<div class="date-summary date-summary-todays-date">',
+            '<h3 class="heading localized-datetime"',
+            'data-datetime="2015-01-02 00:00:00+00:00"',
+            'data-string="Today is {date}"',
+            'data-timezone="America/Los_Angeles"'
+        ]
+        for html in html_elements:
+            self.assertContains(response, html)
 
+    ## CourseStartDate
```

Comment:
This comment could probably stand to be fleshed out a bit (or removed), even if it's just to say "Tests for CourseStartDate."